views/styles/config_styles.js

```diff
@@ -90,4 +90,49 @@ export const configStyles = (theme) => ({
     overflowY: 'visible',
   },
   hiddenInput: { display: 'none' },
+  configFormButtonContainer: { right: 0, bottom: 0, position: 'fixed' },
+  addNewFieldButton: {
+    marginRight: '10px',
+    marginBottom: '5px',
+    marginLeft: '10px',
+    marginTop: '10px',
+    backgroundColor: colors.white,
+  },
+  saveConfigurationButton: {
+    margin: '10px',
+    border: '2px solid rgb(255, 64, 129)',
```

Comment:
Could we store this rgb as a hex and give it a variable?

Reply:
[d7951b8](https://github.com/AMP-SCZ/dpdash/pull/332/commits/d7951b8f581db9d672406104d88aeea54e27d08a)